src/main/java/org/ibp/api/rest/inventory/manager/TransactionResource.java

```diff
@@ -80,6 +83,7 @@
 				"Default sort order is ascending. " +
 				"Multiple sort criteria are supported.")
 	})
+	@PreAuthorize(HAS_MANAGE_TRANSACTIONS + " or hasAnyAuthority('VIEW_TRANSACTIONS', 'CREATE_TRANSACTIONS', 'IMPORT_TRANSACTIONS')")
```

Comment:
@nahuel-soldevilla please remove reference to 'CREATE_TRANSACTIONS', 'IMPORT_TRANSACTIONS'